transaction_service/src/quoteServer/triggers_handler.py

```diff
@@ -0,0 +1,166 @@
+"""
+    Consistenly calls the quote server and triggers collection to execute any buy/sell triggers
+"""
+"""
+    TODO:
+        - add caching to limit amount of calls to quote server
+        - be incorporated into transaction server or organize it so it can be run independently
+"""
```

Comment:
Nice! Will probably need your help to set up our trigger handlers with the Django API

Reply:
Sounds good!